packages/workflow/src/ErrorReporterProxy.ts

```diff
@@ -5,13 +5,16 @@ interface ErrorReporter {
 	report: (error: Error | string, options?: ReportingOptions) => void;
 }
 
+const { NODE_ENV } = process.env;
+const inDevelopment = !NODE_ENV || NODE_ENV === 'development';
+
 const instance: ErrorReporter = {
 	report: (error) => {
 		if (error instanceof Error) {
 			let e = error;
 			do {
 				const meta = e instanceof ApplicationError ? e.extra : undefined;
-				Logger.error(`${e.constructor.name}: ${e.message}`, meta);
+				Logger.error(`${e.constructor.name}: ${e.message}\n${inDevelopment ? e.stack : ''}`, meta);
```

Comment:
not a big issue, but the `\n` will start adding an additional line break in production logs. Maybe we can do this instead:
```suggestion
        if (inDevelopment) console.log(e, meta);
        else Logger.error(`${e.constructor.name}: ${e.message}`, meta);
```